File: langgraph/data/equiv/0.py
from typing import List

def contains_nearby_elements(values: List[float], limit: float) -> bool:
    index = 0
    while index < len(values):
        inner_index = 0
        while inner_index < len(values):
            if index != inner_index:
                gap = abs(values[index] - values[inner_index])
                if gap < limit:
                    return True
            inner_index += 1
        index += 1

    return False

# method name: has_close_elements -> contains_nearby_elements
# parameter name: numbers -> values, threshold -> limit
# changed for loops to while loops
# variable name: idx -> index, idx2 -> inner_index, elem -> values[index], elem2 -> values[inner_index], distance -> gap
# replaced if not condition with direct comparison

File: langgraph/result/qwen/7b-eq2js/0.js
function containsNearbyElements(values, limit) {
    let index = 0;
    while (index < values.length) {
        let innerIndex = 0;
        while (innerIndex < values.length) {
            if (index !== innerIndex) {
                let gap = Math.abs(values[index] - values[innerIndex]);
                if (gap < limit) {
                    return true;
                }
            }
            innerIndex += 1;
        }
        index += 1;
    }

    return false;
}

// method name: hasCloseElements -> containsNearbyElements
// parameter name: numbers -> values, threshold -> limit
// changed for loops to while loops
// variable name: idx -> index, idx2 -> innerIndex, elem -> values[index], elem2 -> values[innerIndex], distance -> gap
// replaced if not condition with direct comparison